client/src/components/Form.js

```diff
@@ -0,0 +1,101 @@
+import PropTypes from 'prop-types'
+import InputField from './InputField'
+import Button from './Button'
+import styled from 'styled-components/macro'
+import { useState } from 'react'
+
+Form.propTypes = {
+  onSubmit: PropTypes.func,
+}
+
+export default function Form({ onSubmit }) {
+  const [isDisabled, setIsDisabled] = useState(true)
+
+  return (
+    <StyledForm
+      onSubmit={handleSubmit}
+      onChange={validateForm}
+      aria-label="create a new user"
+      role="form"
+    >
+      <InputField
+        name="name"
+        labelText="name:"
+        placeholder="Please enter your name..."
+        maxLength="40"
+        minLength="5"
+        required
+      />
```

Comment:
Is it for first name, last name or full name? Maybe that should be clearer. Otherwise the required min length is a bit harsh. 
```suggestion
      <InputField
        name="fullname"
        labelText="full name:"
        placeholder="e.g. Max Mustermann"
        maxLength="40"
        minLength="5"
        required
      />
```